synfig-studio/src/gui/docks/dock_timetrack2.cpp

```diff
@@ -133,15 +137,15 @@ void Dock_Timetrack2::changed_canvas_view_vfunc(etl::loose_handle<CanvasView> ca
 
 		hscrollbar.set_adjustment(canvas_view->time_model()->scroll_time_adjustment());
 
-		update_tool_palette_action();
-		tool_palette.show();
+		update_toolbar_action();
+		set_interp_buttons_visibility(current_widget_timetrack->get_num_waypoints_selected());
 
 		grid.attach(widget_kf_list,            0, 0, 1, 1);
 		grid.attach(widget_timeslider,         0, 1, 1, 1);
 		grid.attach(*current_widget_timetrack, 0, 2, 1, 1);
 		grid.attach(hscrollbar,                0, 4, 2, 1);
 		grid.attach(vscrollbar,                1, 0, 1, 4);
-		grid.attach(tool_palette,              2, 0, 1, 4);
+		grid.attach(*toolbar, 		       2, 0, 1, 4);

```

Comment:
don't use `tab` to align, only for indenting

Reply:
oops, didn't know that, Done